crates/hc_sandbox/src/calls.rs

```diff
@@ -418,64 +417,82 @@ async fn call_inner(client: &mut AdminWebsocket, call: AdminRequestCli) -> anyho
-            msg!("Installed app: {}", app.installed_app_id,);
+            println!("{}", app_info_to_json(app)?);
         }
         AdminRequestCli::UninstallApp(args) => {
             client
                 .uninstall_app(args.app_id.clone(), args.force)
                 .await?;
-            msg!("Uninstalled app: {}", args.app_id,);
+            msg!("Uninstalled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::ListDnas => {
-            let dnas = client.list_dnas().await?;
-            msg!("DNAs: {:?}", dnas);
+            let dnas: Vec<DnaHashB64> = client
+                .list_dnas()
+                .await?
+                .into_iter()
+                .map(|d| d.into())
+                .collect();
+            println!("{}", serde_json::to_value(&dnas)?);
         }
         AdminRequestCli::NewAgent => {
             let agent = client.generate_agent_pub_key().await?;
-            msg!("Added agent {}", agent);
+            println!("{}", serde_json::to_value(agent.to_string())?);
         }
         AdminRequestCli::ListCells => {
-            let cells = client.list_cell_ids().await?;
-            msg!("Cell IDs: {:?}", cells);
+            let cell_id_values: Vec<serde_json::Value> = client
+                .list_cell_ids()
+                .await?
+                .into_iter()
+                .map(serde_json::to_value)
+                .collect::<Result<Vec<serde_json::Value>, serde_json::Error>>()?;
+            let converted = cell_id_values
+                .into_iter()
+                .map(cell_id_json_to_base64_json)
+                .collect::<anyhow::Result<Vec<serde_json::Value>>>()?;
+            println!("{}", serde_json::to_value(&converted)?);
         }
         AdminRequestCli::ListApps(args) => {
             let apps = client.list_apps(args.status).await?;
-            msg!("List apps: {:?}", apps);
+            let values = apps
+                .into_iter()
+                .map(app_info_to_json)
+                .collect::<anyhow::Result<Vec<serde_json::Value>>>()?;
+            println!("{}", serde_json::to_value(values)?);
         }
         AdminRequestCli::EnableApp(args) => {
             client.enable_app(args.app_id.clone()).await?;
-            msg!("Activated app: {:?}", args.app_id);
+            msg!("Enabled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::DisableApp(args) => {
             client.disable_app(args.app_id.clone()).await?;
-            msg!("Deactivated app: {:?}", args.app_id);
+            msg!("Disabled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::DumpState(args) => {
             let state = client.dump_state(args.into()).await?;
-            msg!("DUMP STATE \n{}", state);
+            println!("{}", state);
         }
         AdminRequestCli::DumpConductorState => {
             let state = client.dump_conductor_state().await?;
-            msg!("DUMP CONDUCTOR STATE \n{}", state);
+            println!("{}", state);
```

Comment:
This is coming out pretty printed, is that happening in the conductor? That looks like a bug now that we're making the output consistently unformatted JSON.

Reply:
I can change that in `Conductor::dump_conductor_state()`